internal/data/spicedb.go

```diff
@@ -253,6 +253,10 @@ func (s *SpiceDbRepository) CreateRelationships(ctx context.Context, rels []*api
 	}
 
 	for _, rel := range rels {
+		if !strings.HasPrefix(rel.Relation, "t_") {
```

Comment:
We could pull this out into a function, maybe make the prefix itself a constant too